fdbserver/TLogInterface.h

```diff
@@ -65,6 +65,7 @@ struct TLogInterface {
 	bool operator==(TLogInterface const& r) const { return id() == r.id(); }
 	NetworkAddress address() const { return peekMessages.getEndpoint().getPrimaryAddress(); }
```

Comment:
Although this is not related to this PR, does it make sense (and is it annoying) to change this function name to `primaryAddress()`?

Reply:
Probably should do this in a separate PR.